mod_ci/controllers.py

```diff
@@ -464,22 +465,39 @@ def queue_test(db, gh_commit, commit, test_type, branch="master", pr_nr=0):
     from run import log
 
     fork = Fork.query.filter(Fork.github.like("%/CCExtractor/ccextractor.git")).first()
+    test_case = Test.query.filter(Test.fork_id == Fork.id).first()
```

Comment:
This still won't work. You're currently comparing a fork_id of the Test to any Fork id... That makes no sense. Didn't you intend to use the `fork` object instead of the `Fork` class?

Reply:
But why all the effort to retrieve a test case (which wasn't added yet!) when the pr_nr (in case of a pull request), is passed to the method?